wallet/how-to/use-non-evm-networks/index.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Connect ...
+---
+
+# Connect non-EVM wallets to MetaMask using Snaps
+
+## Non-EVM Wallets
+
+Non-EVM wallets are cryptocurrency wallets that operate on blockchain networks not compatible with the Ethereum Virtual Machine (EVM). 
+Non-EVM wallets can't directly interact with MetaMask or other Ethereum-based applications. 
+Use Snaps to connect these wallets to extend their functionality and integrate them with your existing MetaMask workflow.
+
+## MetaMask Snaps
+
+[MetaMask Snaps](../snaps/learn/about-snaps/) extend MetaMask's functionality. It allows MetaMask to connect with [non-EVM networks]((../snaps/features/non-evm-networks/). 
+
+MetaMask Snaps features include:
+
+- **Expanded network support**: Connect to multiple non-EVM networks.
+- **Custom functionality**: Add new features to MetaMask.
+- **Enhanced security**: Each Snap runs in its own secure environment.
+
+MetaMask Snaps increase interoperability by enabling the use of non-EVM assets with Ethereum-based dapps and provides a unified interface to manage assets in one place.  
```

Comment:
```suggestion
MetaMask Snaps increase interoperability by enabling the use of non-EVM assets and the access to non-EVM network and dapps with a unified interface to consolidating wallet management.
```

We could give an example maybe the cosmos snap : https://snaps.metamask.io/snap/npm/leapwallet/metamask-cosmos-snap/ ? 

In their description the make it clear. By downloading the and installing the snap a user gets access to 50+ dapps in the cosmos ecosystem without having to install a new extension, to store a new password, new keys. Users can do this by simply installing the cosmos snap. This makes the user experience much smoother and consolidates/improves the wallet management.

Reply:
Thanks @khanti42 I think this might be more geared for the user than for developers. I think the plink to the snaps docs above, is probably sufficient. Is that okay?